### PR TITLE
Add manual testing for add delete doctor

### PR DESCRIPTION
**Pull request by ramaven**
### Summary

Add manual testing for adding and deleting doctor

### Checks

- [ ] Link PR to issue
- [ ] Add Reviewer
- [ ] Pass CI

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -819,6 +836,21 @@ testers are expected to do more *exploratory* testing.
 
     6. Other incorrect delete patient commands to try: `delete t/patient`, `delete t/patients`, `delete t/patient x` (where x is an invalid id), etc <br>
        Expected: Error message shown in the feedback box.
+
+1. Deleting a doctor while all doctors are being shown
```
Comment (simonjulianl) [inline docs/DeveloperGuide.md:840]: hem i think the numbering is not 1 ?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -819,6 +836,21 @@ testers are expected to do more *exploratory* testing.
...
+       Ensure at least 1 doctor with id `D001` is there, otherwise please use `add t/doctor` command to add a new doctor.
+       Multiple doctors will be displayed in a table sorted by its id.
+       
+    2. Test case: `delete t/doctor D001`<br>
+       Expected: Doctor with id `D001` is deleted. Details of the deleted doctor are shown in the feedback box.
+
+    3. Test case: `delete t/doctor D001`<br>
+       Expected: No doctor is deleted. Error details shown in the feedback box.
+
+    4. Other incorrect delete doctor commands to try: `delete t/doctor`, `delete t/doctors`, `delete t/doctor x` (where x is an invalid id), `...` <br>
```
Comment (simonjulianl) [inline docs/DeveloperGuide.md:851]: btw use etc. instead of ...

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -780,6 +780,23 @@ testers are expected to do more *exploratory* testing.
...
+    2. Test case: `add t/doctor n/John Smith p/98765432 de/Cardiology`<br>
+       Expected: New doctor whose id `D001` is created, confirmation is shown in feedback box, and the doctor table is shown.
+
+    4. Test case: `add t/doctor n/Tim Burton p/93561345`<br>
+       Expected: Error message of invalid command format is shown in the feedback box, as a department was not specified. 
+
+    5. Test case: `add t/doctor n/Cedric Tom p/11112222333 de/Cardiology`<br>
+       Expected: Error message that phone number should be 8 digits long is shown in the feedback box.
+
+    6. Other incorrect `add t/doctor` commands to try: `add t/doctors`, invalid parameters, `...` <br>
```
Comment (simonjulianl) [inline docs/DeveloperGuide.md:797]: yup here too, use etc. instead of ... (just to make it consistent)

**Review by simonjulianl — CHANGES_REQUESTED** [review]
Minor changes

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (ramaven) [reply docs/DeveloperGuide.md:840]: Oh sorry my bad, fixed.

**Pull request closed without merging**